<h4>Vision</h4>
<p><em><strong>A resource for global interpretation and analysis of HLA polymorphism.</strong></em></p>
<h4>General Ideas</h4>
<p>We have implemented a Public Haplotype Frequency Curation Service (PHYCuS). Our specific goal is to provide a computational resource that will address unmet needs in the field for applications that consume HLA Haplotype Frequency data:</p>
<ul>
<li>standard input formats for genotypes and output formats for haplotypes</li>
<li>standard representations of ambiguity (MAC, GL)</li>
<li>standard validation of HLA</li>
<li>globally unique IDs (GUIDs) to refer to "populations", "cohorts" and the one-to-many relationship between them</li>
<li>access control with appropriate licensing agreements - automated access (REST API not clickthrough pages)</li>
<li>quality metrics</li>
<li>standardized metadata</li>
</ul>
<h4>Future capabiltites of PHYCuS</h4>
<ul>
<li>Global visual of HLA with geographical distributions e.g. <a href="haplostats.org">haplostats.org</a>&nbsp;maps but interactive</li>
<li>Realtime analysis of population differences e.g. <a href="https://nextstrain.org/ncov">nextstrain</a></li>
<li>The ability to seamlessly navigate from HLA haplotypes to alleles to motifs to single amino acid polymorphims or even nucleotide sequence polymorophisms e.g. <a href="https://cran.r-project.org/web/packages/SSHAARP/index.html">SSHAARP</a></li>
</ul>
